topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
+	fmt.Println(ReverseString("Hello World"))
+}
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of runes.
+	str_table := []rune(str)
+
+	// Here we create int that will be a pointer to the front of the str_table.
+	frontIndex := 0
```

Comment:
Let's use. `var frontIndex int` since it is being set to its zero value state.

Reply:
Completed